test/functional/page_objects/common_page.js

```diff
@@ -41,6 +41,7 @@ export function CommonPageProvider({ getService, getPageObjects }) {
     static async navigateToUrlAndHandleAlert(url, shouldAcceptAlert) {
       log.debug('Navigate to: ' + url);
       try {
+        await browser.execute('javascript:localStorage.clear();'); // only needed for IE11, move somewhere?
```

Comment:
I think this should go in webdriver.ts but nothing I tried there worked.  I think it's because the browser isn't launched yet.  Storage *should* be cleared by the options or capabilities but that's not working either.

It's easy to tell that the options and capabilities aren't working from the very first _console test.   That test tries to close the initial Welcome panel.  It fails to click anything for me.  But if you manually close it in IE.  Then kill the test and restart it, the Welcome panel isn't there.  This is because closing the panel is saved in localStorage.

To see this, comment out this browser.execute line which is clearing localStorage.

Reply:
@dmlemeshko worked with me on getting the options working